src/components/dashboard/DashboardPage.jsx

```diff
@@ -95,6 +96,13 @@ const DashboardPage = () => {
         {tabs.map((tab) => React.cloneElement(tab, { key: tab.props.eventKey }))}
       </Tabs>
       <IntegrationWarningModal isEnabled={enterpriseCustomer.showIntegrationWarning} />
+      {/* ExpiredSubscriptionModal is specifically tailored for Verizon learners and is triggered
```

Comment:
nit: we should avoid hardcoding specific enterprise customer's names/slugs/etc. into the code, especially in an `openedx` repo.

```suggestion
      {/* ExpiredSubscriptionModal is specifically tailored for enterprise customers with custom subscription expiration messaging. It is triggered when the learner's license...
```